setup.py

```diff
@@ -14,6 +14,8 @@
         'numpy >= 1.8.0',
         'librosa >= 0.5',
         'tensorflow >= 1.15',
+	'torch',
```

Comment:
I think it's a little extra to ask the users to install `torch` and `torchaudio`. Also, these are only used for the test, so I'd rather put them at https://github.com/keunwoochoi/kapre/blob/master/tox.ini#L14 and Line 21.

Reply:
I know, in fact test works with torch and Librosa... I didn't want to put it on setup but I can remove that part if with Librosa is enough